packages/contentstack-export-to-csv/src/commands/cm/export-to-csv.js

```diff
@@ -258,6 +262,32 @@ class ExportToCsvCommand extends Command {
           }
           break;
         }
+        case config.exportTeams:
+        case 'teams': {
+          try{
+            if (!isAuthenticated()) {
+              this.error(config.CLI_EXPORT_CSV_LOGIN_FAILED, {
+                exit: 2,
+                suggestions: ['https://www.contentstack.com/docs/developers/cli/authentication/'],
+              });
```

Comment:
We could have added this auth check as common rather then duplicating every switch cases.

Reply:
Done